docs/active_storage_migration.md

```diff
@@ -324,6 +324,56 @@ Storage migration is complete this last ticket needs to be reviewed to see if
 the issue has been resolved. This can be done by checking for redraft failures
 due to timeouts in Kibana logs, for example.
 
+## Switchover
+
+> **_NOTE:_** This switchover from paperclip to active storage assumes that checksums have been calculated for the model being activated and the target environment has latest master on it - with the `storage.rake` tasks is available on it.
```

Comment:
```suggestion
> **_NOTE:_** This switchover from paperclip to active storage assumes that checksums have been calculated for the model being activated and the target environment has latest master on it - with the `storage.rake` tasks available on it.
```

Reply:
it needs to be plural "with the `storage.rake` task**s** available on it"

Reply:
I think there is an extra 'is'.

Reply:
oh yeah, friday brain